doc/content/the-things-stack/host/kubernetes/azure/architecture/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Architecture"
+description: ""
+weight: 1
+---
+
+This page describes the architecture of a {{% tts %}} deployment on Azure.
+
+<!--more-->
+
+{{< figure src="azure-infrastructure.svg" alt="Azure Infrastructure" class="plain" >}}
+
+## Networking
+
+The main infrastructure is contained inside a Virtual Network, grouped into separate Private Subnets. To communicate with the internet we use a Network Address Translation Gateway (NAT GW). Internal load balancer (LB) is used to handle incoming traffic.
+
+The NAT GW is assigned a Public IP Address Prefix while the LB is assigned a static Public IP Address.
+
+## Persistence
+
+{{% tts %}} relies on two databases: PostgreSQL and Redis. Azure offers these databases as managed services, so that we can focus on operating a LoRaWAN® network instead of operating databases.
```

Comment:
Remove `so that we can focus on operating a LoRaWAN® network instead of operating databases.`